wikimedia/wikipedia-ios
# Talk pages - Push to talk page revision history

**Phabricator:** 
https://phabricator.wikimedia.org/T310288
https://phabricator.wikimedia.org/T310294

### Notes
This pushes to our revision history screen from the history buttons in the talk page toolbar and overflow menu.

### Test Steps
1. Tap on the revision history button in the toolbar. Confirm it pushes to our native history page.
2. Tap on the revision history button in the overflow menu. Confirm it pushes to our native history page.
3. Repeat steps 1 & 2 in a different language.
